superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -113,6 +125,11 @@ export default function DataSourcePanel({
     columns,
     metrics,
   });
+  const [showAllMetrics, setShowAllMetrics] = useState(false);
+  const [showAllColumns, setShowAllColumns] = useState(false);
+
+  const DEFAULT_MAX_COLUMNS_LENGTH = 50;
+  const DEFAULT_MAX_METRIC_LENGTH = 50;
```

Comment:
Biggest nitpick ever. 
```suggestion
  const DEFAULT_MAX_METRICS_LENGTH = 50;
```